.gitignore

```diff
@@ -21,9 +15,4 @@ local.properties
 bin/
 gen/
 out/
-captures/
-lint/intermediates/
-lint/generated/
-lint/outputs/
-lint/tmp/
-
+captures/
```

Comment:
Please check if rest of the directories can be removed as well

Reply:
Done